src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,13 @@
+$sell-size:100px;
+$gap:1px;
+$calendar-width: 7 * $sell-size + 6 * $gap;
+$base-color: #eee;
+
+$days:
+  "mon" 0,
+  "tue" $sell-size + $gap,
+  "wed" 2 * $sell-size + 2 * $gap,
+  "thu" 3 * $sell-size + 3 * $gap,
+  "fri" 4 * $sell-size + 4 * $gap,
+  "sat" 5 * $sell-size + 5 * $gap,
+  "sun" 6 * $sell-size + 6 * $gap;
```

Comment:
to avoid repeating this formula for each day set it once in the loop where you apply this variable